src/ontology/mondo-edit.obo

```diff
@@ -513271,6 +513299,7 @@ subset: rare
 synonym: "IDDSELD" RELATED ABBREVIATION []
 synonym: "INTELLECTUAL DEVELOPMENTAL DISORDER WITH SEIZURES AND LANGUAGE DELAY" RELATED []
 synonym: "SETD1B-NDD" EXACT ABBREVIATION [https://orcid.org/0000-0001-9310-0163]
+synonym: "SETD1B-Related Neurodevelopmental Disorder" EXACT []
```

Comment:
This synonym was added without an x-ref. Also, it is the same as the synonym below, but with different capitalization. We should ignore capitalization except for Gene names.